readthedocs/search/parsers.py

```diff
@@ -153,9 +153,40 @@ def _parse_sections(self, title, body):
                         'title': title,
                         'content': content,
                     }
+                except Exception as e:
+                    log.info("Unable to index section.", section=str(e))
+
+        dls = body.css("dl")
+        for dl in dls:
+            dts = dl.css("dt")
```

Comment:
We only want to index `dt`s that have an ID, otherwise we won't be able to link to them, results would be distracting (they will be indexed in the section, so we aren't losing any data).